CONTRIBUTING.md

```diff
@@ -143,10 +143,10 @@ The process is adopted from the Thanos community.
 
 * **Owners:**
   * `<@author: single champion for the moment of writing>`
-
+* **Status:**
+  * `<Accepted/ToDo/In-Progress/Implemented>`
```

Comment:
I'd suggest
* Accepted (the default status)
* Rejected (in case a feature request doesn't get accepted but we want to document why).
* Implemented

The in-progress status might be more complicated to maintain (e.g. do we want to revert the status if the work stalls?). It's also easier to follow in the GitHub issue. The same remark would apply to `ToDo` IMHO.